### ISSUE TITLE
SRDB-V5 to do list

### ISSUE THREAD

**Issue by jinshijian** [bpbond/srdb#78]
### Things have to be finished before releasing V5
* Updating papers until 2017 (working)
* Including Russian papers
* Unique ID for every row (working)
* Connect MGRsD and HGRsD with SRDB (done)
* Separate "Rs equation" section out and collect equations for the "new" added papers
* Add study_number, middle_year, and site_ID column in the equation file?
* Adding collar_depth and measure_time information
* Standardize __Manipulation__ (done)

### Things can be done (but need more time)
* Add SOC% column and collect SOC% information (Not now)
* Collect "Rh equation and Q10 for Rh/Ra" information
* Updating MGRsD and HGRsD
* Goal: manuscript submitted by April 1st
* From Emma: include replication (e.g., number of collars), something about the low representability of a single measurement within a larger area
* versions of the database need to have individual DOIs for version tracking and proper citation; metadata needs to be revisited to better describe original studies and SRDB improvements; SRDB needs to be interoperable with data from other networks (e.g., AmeriFlux, ISCN, WoSIS).
* There is a need to improve internet-based hosting services that give us the ability to build a 21st-century data set that is modified by, and grows with, the needs of the scientific community. 
* Number of samples within how large a study area
* Broad-leaf or needle leaf

**Comment by bpbond** [comment]
A question: do we _want_ MGRsD and HDRsD in the same repository as SRDB? Advantages: "one stop shop" for researchers; easier to keep `Site_ID` consistent. Disadvantages: more cumbersome; do we have to update everything in sync?

**Comment by bpbond** [comment]
We decide too difficult to get EVERYTHING updated by April 1. Let's keep separate for now.

**Comment by bpbond** [comment]
* Contact ORNL and let them know preparing v5

**Comment by bpbond** [comment]
@jinshijian please see and respond/resolve
- #85 
- #88 

Looking through the list above,
>Separate "Rs equation" section out and collect equations for the "new" added papers

This still needs to be done for all the papers, right? Maybe we hold off for now though, I'm thinking, and focus on finishing up PR #86 so you can finish your ESSD draft.

**Comment by jinshijian** [comment]
Check the blank measure method

**Comment by bpbond** [comment]
@jinshijian I feel like we need to have a definite "stop" point for v5. After standardizing the `Manipulation` ( #96 ), `Meas_method`, and `Partition_method` fields, and incorporating Nazar's data, is that the end? (I think so?)

**Comment by bpbond** [comment]
Closed by #106  woo hoo!
